qa/multi-cluster-search/src/test/resources/rest-api-spec/test/remote_cluster/10_basic.yml

```diff
@@ -50,6 +50,7 @@
         body:
           settings:
             index:
+              number_of_shards: 1
```

Comment:
Not sure this change is in the right yaml test. The correspoinding @Awaitsfix for #58345 seems to be in multi-cluster-search/src/test/resources/rest-api-spec/test/multi_cluster/80_index_name_agg.yml

Reply:
This part defines the remote indices that are used in the tests. See https://github.com/elastic/elasticsearch/blob/master/qa/multi-cluster-search/src/test/resources/rest-api-spec/test/multi_cluster/80_index_name_agg.yml#L36